elasticsearch/README.md

```diff
@@ -73,6 +73,10 @@ specific resources deployed:
   - One GCE VM for each data node
   - One Kubernetes Pod for each data VM
 
+The nodes are created on a Kubernetes cluster on a private cluster, meaning 
+the nodes have reserved IP addresses only. This is useful so large clusters 
+do not run quota issues for external IP addresses.
```

Comment:
Add links to:
https://cloud.google.com/kubernetes-engine/docs/concepts/private-cluster-concept
https://cloud.google.com/compute/quotas#ip_addresses

s/do not run quota issues/do not run into quota issues/